sdk/servicebus/Microsoft.Azure.WebJobs.Extensions.ServiceBus/api/Microsoft.Azure.WebJobs.Extensions.ServiceBus.netstandard2.0.cs

```diff
@@ -81,6 +81,7 @@ public partial class ServiceBusOptions : Microsoft.Azure.WebJobs.Hosting.IOption
         public System.TimeSpan MaxAutoLockRenewalDuration { get { throw null; } set { } }
         public System.TimeSpan MaxBatchWaitTime { get { throw null; } set { } }
         public int MaxConcurrentCalls { get { throw null; } set { } }
```

Comment:
Any chance we can reuse `MaxConcurerntCalls` rather than introducing a new setting?

Reply:
No, that one is for non-session configuration. For sessions, we have MaxConcurrentSessions and the new setting MaxConcurrentCallsPerSession, which maps to the properties in the SDK.